letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -80,6 +84,17 @@ if [ $BASENAME = "letsencrypt-auto" ]; then
   HELP=0
 fi
 
+# Support for busybox and others where there is no "command",
+# but "which" instead
+if command -v command > /dev/null 2>&1 ; then
+  export WHERE_IS="command -v"
```

Comment:
Maybe call this `EXISTS` rather than `WHERE_IS` since we're using it to see if the command exists at all, not to find a path?